solenoid/urls.py

```diff
@@ -22,24 +22,29 @@
 from .views import HomeView
 
 urlpatterns = [
-    re_path(r'^admin/', admin.site.urls),
-    re_path(r'^$', HomeView.as_view(), name='home'),
-    re_path(r'^celery-progress/', include('celery_progress.urls',
-                                          namespace='celery_progress')),
-    re_path(r'^records/', include('solenoid.records.urls',
-                                  namespace='records')),
-    re_path(r'^emails/', include('solenoid.emails.urls', namespace='emails')),
-    re_path(r'^people/', include('solenoid.people.urls', namespace='people')),
-    re_path(r'^oauth2/', include('social_django.urls', namespace='social')),
-    re_path(r'^logout/$',
-            SolenoidLogoutView.as_view(template_name='userauth/logout.html'),
-            name='logout'),
-    re_path(r'^500/$', server_error),
-    ]
+    re_path(r"^admin/", admin.site.urls),
+    re_path(
+        r"^accounts/",
+        include("solenoid.accounts.urls", namespace="accounts"),
+    ),
+    re_path(r"^$", HomeView.as_view(), name="home"),
+    re_path(
+        r"^celery-progress/", include("celery_progress.urls", namespace="celery_progress")
+    ),
+    re_path(r"^records/", include("solenoid.records.urls", namespace="records")),
+    re_path(r"^emails/", include("solenoid.emails.urls", namespace="emails")),
+    re_path(r"^people/", include("solenoid.people.urls", namespace="people")),
+    # re_path(r'^oauth2/', include('social_django.urls', namespace='social')),
+    # re_path(r'^logout/$',
+    #         SolenoidLogoutView.as_view(template_name='userauth/logout.html'),
+    #         name='logout'),
```

Comment:
Do you think there is value in having `/logout` (confirmed does not work) redirect to view `/accounts/logout` (confirmed works)?   Not a request, just an option.

Reply:
Thank you for the suggestion! The change has been made.